23_dedicated_keylime_user_account.md

```diff
@@ -0,0 +1,230 @@
+components under the Keylime user. This may be as simple as modifying the
+systemd unit file to specify a user under which to run Keylime.
+
+Other files that will need to be modified to take this change into account are
+`installer.sh`, `run_tests.sh` as well as the Ansible role for Keylime. There
+may be some work to do on the SElinux labels too. 
+
+Finally, the documentation will have to be updated as well.
+
+### User Stories (optional)
+
+#### Story 1
+
+As a system administrator in an organisation with strong security rules
+regarding the software that may be run, and how, i would like to use Keylime for
+the benefits it provides, but cannot as only a very limited subset of software
+from a pre-approved list is allowed to run as root on the organisation's hosts,
+and Keylime is not one of them.
+
+Now that Keylime uses its own dedicated user account, this aligns Keylime with my
+organisation's policy and i can deploy Keylime.
+
+
+### Notes/Constraints/Caveats (optional)
+
+TBD
+<!--
+What are the caveats to the proposal?
+What are some important details that didn't come across above.
+Go in to as much detail as necessary here.
+This might be a good place to talk about core concepts and how they relate.
+-->
+
+### Risks and Mitigations
+
+TBD
+<!--
+What are the risks of this proposal and how do we mitigate.  Think broadly.
+For example, consider both security and how this will impact the larger
+enhancement ecosystem.
+
+How will security be reviewed and by whom?
+-->
+
+## Design Details
+
+TBD
+<!--
+This section should contain enough information that the specifics of your
+change are understandable.  This may include API specs (though not always
+required) or even code snippets.  If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+
+### Test Plan
+
+TBD
+<!--
+**Note:** *Not required until targeted at a release.*
+
+Consider the following in developing a test plan for this enhancement:
+- Will there be e2e and integration tests, in addition to unit tests?
+- How will it be tested in isolation vs with other components?
+
+No need to outline all of the test cases, just the general strategy.  Anything
+that would count as tricky in the implementation and anything particularly
+challenging to test should be called out.
+
+All code is expected to have adequate tests (eventually with coverage
+expectations).
+-->
+
+### Upgrade / Downgrade Strategy
+
+TBD
+<!--
+If applicable, how will the component be upgraded and downgraded? Make sure
+this is in the test plan.
+
+Consider the following in developing an upgrade/downgrade strategy for this enhancement
+-->
+
+## Drawbacks
+
+It makes the overall design of Keylime a bit more complicated. However, we
+believe that benefits provided by improving privilege separation vastly
+outweigh this drawback.
+
+## Alternatives
+
+Privilege separation (and more generally, defense in depth) can be done at
```

Comment:
Would containerization be an acceptable alternative as well (assuming SPC isn't required)?